README.md

```diff
@@ -122,25 +122,18 @@ Within the following table, we summarized the current NNI capabilities, we are g
             <li><a href="https://nni.readthedocs.io/en/stable/Tuner/BuiltinTuner.html#SMAC">SMAC</a></li>
             <li><a href="https://nni.readthedocs.io/en/stable/Tuner/BuiltinTuner.html#MetisTuner">Metis Tuner</a></li>
             <li><a href="https://nni.readthedocs.io/en/stable/Tuner/BuiltinTuner.html#GPTuner">GP Tuner</a></li>
+            <li><a href="https://nni.readthedocs.io/en/stable/Tuner/BuiltinTuner.html#DNGOTuner">DNGO Tuner</a></li>
             </ul>
-          <b>RL Based</b>
-          <ul>
-            <li><a href="https://nni.readthedocs.io/en/stable/Tuner/BuiltinTuner.html#PPOTuner">PPO Tuner</a> </li>
-          </ul>
         </ul>
-          <a href="https://nni.readthedocs.io/en/stable/NAS/Overview.html">Neural Architecture Search</a>
+          <a href="https://nni.readthedocs.io/en/stable/NAS/Overview.html">Neural Architecture Search (Retiarii)</a>
```

Comment:
Retiarii for Neural Architecture Search

Reply:
just to double check, these links are the refactored links for Retiarii, right?

Reply:
yes, they are the links for Retiarii, but they will not work until 2.3 is released.

Reply:
> Retiarii for Neural Architecture Search

I like this statement when it is at the home page, because it is a little strange, it is confusing that what is the relation between NNI and Retiarii...
Maybe we can directly remove "(Retiarii)"?